.github/actions/to-integration-tests/cdn.json

```diff
@@ -1,8 +1,15 @@
 {
+	"client_certificate_authentication" : {
+		"root_certificates_directory" : "/etc/pki/tls/traffic_ops/"
+	},
 	"disable_auto_cert_deletion": false,
 	"use_ims": true,
 	"role_based_permissions": true,
 	"traffic_ops_golang": {
+		"tls_config": {
+			"MinVersion": 769,
```

Comment:
please in the name of all that is good and sacred in this world tell me that we don't seriously use hard-coded, implementation-defined, numeric constants to define security-related configuration

Reply:
This was suggested before in https://github.com/apache/trafficcontrol/pull/4573#discussion_r400346222:

> 769, 770, etc are not very intuitive -- would it be better to use a string or float that we translate into `tls.VersionTLS11` from the `tls` package in the stdlib? If 770 is basically required in order for TO to even communicate w/ Riak, should we just make 770 the default, allowing people to change it if they know what they're doing?

Reply:
I really thought we took that advice back then. I'd have been pushing for it all this time if I was more aware of my surroundings

Reply:
Changing that is a little outside the scope of this PR though, so a new issue should be opened for it.

Reply:
Of course. I was just hoping he was doing it wrong.